demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -239,3 +242,58 @@ def is_valid_file_path(self):
         if not is_valid_path:
             print_error(Errors.invalid_file_path(self.file_path))
         return is_valid_path
+
+    def print_errors(self, err):
+        """A rapper which runs the print error message for a list of errors in yaml
+        Returns:
+            parsed error message from pykwalify
+        """
+        if ".\n" in str(err):
+            for error in str(err).split('.\n'):
+                self.print_error_msg(error)
+        else:
+            self.print_error_msg(str(err))
+
+    def print_error_msg(self, err):
+        """Returns a parsed error message from pykwalify
+        Args: an schema error message from pykwalify
+        """
+        path_from_error = str(err).split('Path: ')[1][2:-4].split('/')
+        if path_from_error[0] != '':
+            curr = self.current_file
+            key_from_error = str(err).split('key')[1].split('.')[0].replace("'", '-').split('-')[1]
```

Comment:
maybe comment an error for an example. also, what happens if the split/replace returns a value/attribute error?
try/catch please.

Reply:
There is an try and catch when the function is called, and if an error occurs we return the pykwalify response unparsed